app/src/scenes/AddDrink/ConsosList.js

```diff
@@ -110,22 +110,26 @@ const ConsosList = ({ navigation, route }) => {
       const kcal = drinkFromCatalog.kcal;
       const price = drinkFromCatalog.price;
       const volume = drinkFromCatalog.volume;
-      const response = await API.post({
-        path: '/consommation',
-        body: {
-          matomoId: matomoId,
-          id: drink.id,
-          name: drink.displayDrinkModal,
-          drinkKey: drink.drinkKey,
-          quantity: Number(drink.quantity),
-          date: makeSureTimestamp(addDrinkModalTimestamp),
-          doses: doses,
-          kcal: kcal,
-          price: price,
-          volume: volume,
-        },
-      });
-      if (response.showNewBadge || response.showInAppModal) showToast = false;
+      try {
+        const response = await API.post({
+          path: '/consommation',
+          body: {
+            matomoId: matomoId,
+            id: drink.id,
+            name: drink.displayDrinkModal,
+            drinkKey: drink.drinkKey,
+            quantity: Number(drink.quantity),
+            date: makeSureTimestamp(addDrinkModalTimestamp),
+            doses: doses,
+            kcal: kcal,
+            price: price,
+            volume: volume,
+          },
+        });
+        if (response.showNewBadge || response.showInAppModal) showToast = false;
```

Comment:
pas besoin du try catch ici

```suggestion
        if (response?.showNewBadge || response?.showInAppModal) showToast = false;
```